src/util.ts

```diff
@@ -91,6 +91,13 @@ export function clearCachedValues (movie: Movie): void {
   valCache.delete(movie)
 }
 
+type InterpolateType = <U = number | object>(
```

Comment:
Normally, we like to only make one change at a time, but since we're now making this type visible, can we rename it to something more concise (for example, `Interpolate`)? Not sure why I named it `InterpolateType` in the first place 😂

Reply:
Alright, I'll go with `Interpolate`!
So, in conclusion the last part of this PR will rename the type... and address the tests in future PR right ?

Reply:
Yeah, let's address the tests later, once the rest parameter's type is fixed.